cliquet/views/hello.py

```diff
@@ -13,6 +13,7 @@ def get_hello(request):
     data = dict(
         hello=project_name,
         version=settings['project_version'],
+        protocol_version=API_VERSION,
```

Comment:
That is exactly what I had in mind. What if we break the Cliquet python API, but keep the HTTP API identical? I think it could a separated constant, no?